src/hypothesis/extra/pytestplugin.py

```diff
@@ -19,6 +19,9 @@
 import pytest
 
 PYTEST_VERSION = tuple(map(int, pytest.__version__.split('.')[:3]))
+LOAD_PROFILE_OPTION = '--hypothesis-profile'
+
+PYTEST_VERSION = tuple(map(int, pytest.__version__.split('.')))
```

Comment:
Yeah, I noticed this right after merging. It's now fixed in master. I think this was a hiccup caused by the last rebase.